test/integration/test_seq_copy_int.py

```diff
@@ -80,7 +80,7 @@
      "--beam-size 2 --beam-search-stop first",
      True, 3),
     # Basic transformer with LHUC
-    ("--encoder transformer --decoder {decoder}"
+    ("--encoder transformer --decoder transformer"
```

Comment:
Oh ok, so we'd just run it twice with Transformer.

Reply:
Yes, seems like the easiest solution for now